content/en/docs/tasks/inject-data-application/define-command-argument-container.md

```diff
@@ -1,66 +1,66 @@
 ---
-title: Define a Command and Arguments for a Container
+title: Defining a Command and Arguments for a Container
 content_template: templates/task
 ---
 
 {{% capture overview %}}
 
 This page shows how to define commands and arguments when you run a container
-in a {{< glossary_tooltip term_id="pod" >}}.
+in a Kubernetes Deployment.
 
-{{% /capture %}}
+{{% endcapture %}}
```

Comment:
I think the captures and endcaptures need to be reverted for Hugo too, but I am not 💯 so....

/assign @chenopis 
/assign @bep